dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.js

```diff
@@ -167,13 +167,13 @@ const findAmendmentByStatusAndFacilityId = async (facilityId, status) => {
     try {
       const collection = await db.getCollection('tfm-facilities');
       const amendment = await collection
-        .aggregate([ // TODO SR-8
-          { $match: { _id: ObjectId(facilityId) } },
+        .aggregate([
+          { $match: { _id: { $eq: ObjectId(facilityId) } } },
           { $unwind: '$amendments' },
-          { $match: { 'amendments.status': status } },
-          { $project: { _id: 0, amendments: 1 } },
+          { $match: { 'amendments.status': { $eq: status } } },
+          { $project: { _id: false, amendments: true } }, // TODO SR-8 Changed values in project to true or false to better represent the functionality
```

Comment:
Remove todo once noted:
```suggestion
          { $project: { _id: false, amendments: true } },
```